drivers/audio/Kconfig

```diff
@@ -55,4 +56,28 @@ source "drivers/audio/Kconfig.mpxxdtyy"
 
 endif # AUDIO_DMIC
 
+menuconfig PDM
+	bool "PDM (Pulse Density Modulation) Drivers"
+	help
+	  Enable config options for PDM drivers.
+
+if PDM
+
+config PDM_BUFFER_SIZE
+	int "Size of the buffer for the PDM driver"
+	range 10 32767
+	default 256
+	depends on PDM
+	help
+		Size of the PDM buffer,
+		can be between 10-32767.
```

Comment:
This is actually the number of samples that the driver is to pass to the application in one block (in one call to the provided data handler). It should be rather a parameter of the `pdm_start` function, or a field in the `pdm_config` structure.

Reply:
I see what you mean, but the driver must have two internal buffers passing to the HAL driver, in a ping-pong manner. I could use `malloc()` in order to have the size of the buffer in the `pdm_config` struct, but doesn't it make more sense to do it as a Kconfig option?